src/ui_address.c

```diff
@@ -65,6 +65,11 @@ void
 step_address()
 {
   char checksummed[65];
+  uint8_t publicKey[32];
+  cx_ecfp_private_key_t privateKey;
+
+  algorand_key_derive(0, &privateKey);
```

Comment:
Address only on accountId 0 even thoug multiple accounts are supported?

Reply:
Kept unchanged for now.